src/modules/sensors/voted_sensors_update.cpp

```diff
@@ -702,14 +702,16 @@ bool VotedSensorsUpdate::checkFailover(SensorData &sensor, const char *sensor_na
 						      ((flags & DataValidator::ERROR_FLAG_HIGH_ERRDENSITY) ? " ERR DNST" : ""));
 
 				// reduce priority of failed sensor to the minimum
-				sensor.priority[failover_index] = 1;
+				sensor.priority[failover_index] = ORB_PRIO_MIN;
 
 				PX4_ERR("Sensor %s #%i failed. Reconfiguring sensor priorities.", sensor_name, failover_index);
 
 				int ctr_valid = 0;
 
 				for (uint8_t i = 0; i < sensor.subscription_count; i++) {
-					if (sensor.priority[i] > 1) { ctr_valid++; }
+					if (sensor.priority[i] > 1) {
```

Comment:
You don't want to use `ORB_PRIO_MIN` here?

Reply:
Yes I suppose so. Unrelated to this PR I came across a case where this was stuck spamming continuously on a failure, so I was actually planning to take a closer look soon.